parmed/scripts.py

```diff
@@ -171,13 +171,18 @@ def _print_warnings(message, category, filename, lineno, file=None, line=None):
                 for line in f:
                     if line.startswith('# Log started on'): continue
                     readline.add_history(line.rstrip())
-            logfile = open(opt.logfile, 'a')
-            now = datetime.datetime.now()
-            logfile.write('# Log started on %02d/%02d/%d [mm/dd/yyyy] at '
-                          '%02d:%02d:%02d\n' % (now.month, now.day, now.year,
-                                            now.hour, now.minute, now.second))
-            parmed_commands.setlog(logfile)
-            close_log_file = True
+            try:
+                logfile = open(opt.logfile, 'a')
+                now = datetime.datetime.now()
+                logfile.write('# Log started on %02d/%02d/%d [mm/dd/yyyy] at '
+                              '%02d:%02d:%02d\n' % (now.month, now.day, now.year,
+                                                now.hour, now.minute, now.second))
+                parmed_commands.setlog(logfile)
+                close_log_file = True
+                could_open_logfile = True
+            except:
+                print ("could not open logfile. Skip logging")
+                could_open_logfile = False
```

Comment:
Why not get rid of `could_open_logfile` and just set `close_log_file = False` here instead?

Also, can you print `Could not open logfile. Skip logging` to `stderr instead of stdout?